### ISSUE TITLE
Add Battery Voltage Cell Difference

### ISSUE THREAD

**Issue by Salada71** [wills106/homeassistant-solax-modbus#1041]
### Description

Add new sensor.
I do not know how to push to branch. I am not programmer.

### Suggested Solution

const.py:
```
def value_function_battery_voltage_cell_difference(initval, descr, datadict):
    return  datadict.get('cell_voltage_high', 0) - datadict.get('cell_voltage_low',0)
```

plugin_solax.py:
```
SolaXModbusSensorEntityDescription(
        name = "Battery Voltage Cell Difference",
        key = "battery_voltage_cell_difference",
        value_function= value_function_battery_voltage_cell_difference,
        native_unit_of_measurement = UnitOfElectricPotential.VOLT,
        device_class = SensorDeviceClass.VOLTAGE,
        state_class = SensorStateClass.MEASUREMENT,
        rounding = 3,
        allowedtypes = AC | HYBRID | GEN4 | GEN5,
    ),
```

**Comment by wills106** [comment]
I have released [2024.09.1](https://github.com/wills106/homeassistant-solax-modbus/releases/tag/2024.09.1) with this addition.

**Comment by wills106** [comment]
Closing as added in change.
